### DISCUSSION TITLE
git checkout help

### DISCUSSION THREAD

**Discussion by SimpleCppCoder** [ocornut/imgui#5751] [New Users Build/Link/Run issues ONLY!]
Hello! I'm added ImGui to my project, I git cloned the repo into my project but I can't seem to checkout the latest version?
Right now I have the latest files but I would like to get the 1.88 version of the files. I have tried the following:

- `git checkout 9aae45eb4a05a5a1f96be1ef37eb503a12ceb889` with the string being the last 1.88 commit
- `git checkout tags/v1.88`
- `git checkout v1.88`

But none of them worked. Is there some way I can checkout the 1.88 version of the files instead of downloading from the github releases?

**Comment by PathogenDavid** [comment]
All three of those should work, please clarify what you mean by "none of them worked".
